pkg/controller/pv-monitor-controller/event_handler.go

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Do we need a sub-folder "pv-monitor-controller" under "controller"?  Should the files just be directly under "controller"?

Reply:
just in case we want to add new controllers

Reply:
We'll change the directory structure when there's a need for new controllers.
For now, please remove extra layer of directory,

Reply:
ok, done